src/TimeZone.php

```diff
@@ -53,45 +54,46 @@ abstract public function getId() : string;
      *
      * @return int The offset from UTC in seconds.
      */
-    abstract public function getOffset(Instant $pointInTime) : int;
+    abstract public function getOffset(Instant $pointInTime): int;
 
-    public function isEqualTo(TimeZone $other) : bool
+    public function isEqualTo(TimeZone $other): bool
     {
         return $this->getId() === $other->getId();
     }
 
-    public function __toString() : string
-    {
-        return $this->getId();
-    }
-
     /**
      * @deprecated please use fromNativeDateTimeZone instead
      */
-    public static function fromDateTimeZone(\DateTimeZone $dateTimeZone) : TimeZone
+    public static function fromDateTimeZone(DateTimeZone $dateTimeZone): TimeZone
     {
         return self::fromNativeDateTimeZone($dateTimeZone);
     }
 
-    public static function fromNativeDateTimeZone(\DateTimeZone $dateTimeZone) : TimeZone
+    public static function fromNativeDateTimeZone(DateTimeZone $dateTimeZone): TimeZone
     {
         return TimeZone::parse($dateTimeZone->getName());
     }
 
     /**
      * Returns an equivalent native `DateTimeZone` object for this TimeZone.
+     *
      * @deprecated please use toNativeDateTimeZone instead
```

Comment:
Is there a way to force each `@param`, `@return`, `@throws`, `@deprecated` sentence to end with a punctuation mark?

Reply:
Forcing no. However I removed https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/rules/phpdoc/phpdoc_annotation_without_dot.rst to allow it.

Reply:
I eventually found a rule that forces the dot on `@param` and `@throws` annotations. It also cannot be automatically fixed. That's better than nothing I guess.